docs/installation.md

```diff
@@ -0,0 +1,123 @@
+```sh
+# Install brew
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+# Install AsyncAPI CLI
+brew install asyncapi
+```
+
+#### `pkg` installation
+Every release of the AsyncAPI CLI has its own macOS dedicated `pkg` file that enables you to install the CLI tool as a macOS application. 
+To download the latest CLI release, run this command in your terminal:
+```sh
+curl -OL https://github.com/asyncapi/cli/releases/latest/download/asyncapi.pkg
+```
+
+To download a specific CLI release, run this command in your terminal:
+```sh
+curl -OL https://github.com/asyncapi/cli/releases/download/<replace this with the specific CLI version e.g v0.13.0>/asyncapi.pkg
+```
+
+<Remember>
+Follow this link for all <a href="https://github.com/asyncapi/cli/releases">AsynAPI CLI releases</a>.
+</Remember>
+
+After downloading the AsyncAPI CLI, install it via the following command:
+
+```sh
+sudo installer -pkg asyncapi.pkg -target /
+```
+
+### Linux
+Selecting the appropriate AsyncAPI CLI installation method on a Linux operating system depends on your Linux distro.
+
+#### Debian based distros
+For Debian based distros, you can install the AsycAPI CLI using the `dpkg` package manager for Debian.
+```sh
+curl -OL https://github.com/asyncapi/cli/releases/latest/download/asyncapi.deb
+```
+
+To download a specific release of the CLI, run this command in your terminal:
+```sh
+curl -OL https://github.com/asyncapi/cli/releases/download/<replace this with the specific CLI version e.g v0.13.0>/asyncapi.deb
+```
+
+#### Other distros
+You can install the AsyncAPI CLI for other Linux distros using the archive `tar.gz` file. To download the latest release of the CLI, run this command in your terminal:
+```sh
+curl -OL https://github.com/asyncapi/cli/releases/latest/download/asyncapi.tar.gz
+```
+
+To download a specific release of the CLI, run this command in your terminal:
+```sh
+curl -OL https://github.com/asyncapi/cli/releases/download/<replace this with the specific CLI version e.g v0.13.0>/asyncapi.tar.gz
+```
+
+Once you have downloaded the archived file, untar it by running this command in your terminal:
+```sh
+tar -xzf asyncapi.tar.gz
+```
+
+The step above will create an `AsynAPI` directory in the current path. To run the CLI from anywhere, you must create a `symlink`. If the current path you are on is `/user/local/bin`, for example, you must create the `symlink` in the `/user/local/bin` directory by following these steps:
+```sh
+# cd into the unarchived directory
+cd asyncapi
+
+# get the absolute path
+pwd
+
+# Create a symlink
+ln -s <absolute-path>/bin/asyncapi /user/local/bin/asyncapi
+
+# The "asyncapi" command should be available to be used
+asyncapi
+```
+
+---
+
+<DocsButton
+ suggestions={[
+    {
+      href:'/docs/introduction',
+      type: 'back',  
+      title: 'Introduction',
+   },
+   {
+      href:'/docs/usage',
+      type: 'next',  
+      title: 'Usage',
+   }
+ ]}
+/>
```

Comment:
```suggestion

```

1. Yo, [we talked about this... I told you we don't need the prev/next buttons in the `CLI` repo](https://github.com/asyncapi/website/pull/1097#issuecomment-1338524232); you will need to add prev/next buttons to the NEXT PR once we have the automation to copy over this content over the `/website` repo. Please confirm if this makes sense. 😜 ✌🏽
1. The routes are wrong because we need routes from the `/website` repo pages perspective, based on the actual content buckets and sections we have in AsyncAPI Docs.